analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -343,6 +343,8 @@ private fun resolveLocalSpec(definitionFile: File): File? =
 private fun getIdentifier(name: String, version: String) =
     Identifier(type = "NuGet", namespace = "", name = name, version = version)
 
+private fun NuGetDependency.getId() = Identifier("NuGet::$name:$version")
```

Comment:
This should probably call the above `getIdentifier()` to not hard-code "NuGet" more than needed.

Reply:
Done in a separate commit on top.

Reply:
Why only on top? It's really inconvenient to come across this again in incremental reviews. Do you want to bump your commit stats? 😉

Reply:
It just follows your rules to do one thing at a time. It's a very minor issue. In my view you are overreacting to this nit. I prefer to keep it still.